maps/offmap_vr/talon/talon2.dmm

```diff
@@ -721,6 +721,24 @@
 	},
 /turf/simulated/floor/tiled/eris/steel,
 /area/talon/decktwo/central_hallway)
+"bG" = (
+/obj/structure/cable/green{
+	d1 = 1;
+	d2 = 2;
+	icon_state = "1-2"
+	},
+/obj/structure/catwalk,
+/obj/machinery/alarm/talon{
```

Comment:
...Why are the tether anomaly lab air alarms being used for talon?

Reply:
That is a very good question.